src/data/STATUSES/root/RDM.ts

```diff
@@ -44,7 +44,7 @@ export const RDM = ensureStatuses({
 		id: 1971,
 		name: 'Manafication',
 		icon: iconUrl(17491),
-		duration: 15000,
+		duration: 30000,
```

Comment:
This definitely needs to be done in the layers file so that it doesn't effect pre-7.05 log timelines.  Also, shouldn't this be for the Magicked Swordplay status and not the Manification status?

Reply:
It would seem when 7.01 came out I mistakenly hit magicked swordplay instead of manafication.  Also what do you mean layers, changing the duration on a status is done via its duration properties is it not?

Reply:
There are files in the layers directory (there's a corresponding one for STATUSES, but this one is for ACTIONS: https://github.com/xivanalysis/xivanalysis/blob/dawntrail/src/data/ACTIONS/layers/patch7.05.ts) that contain just the changes to given actions or statuses in a specific patch.  The layers files allows us to support both pre- and post-patch logs, since the root data values are as of 7.0, and then we can add each subsequent patch's changes from the layers files.  

Which is where the Manafication change from 7.01 was already previously made: https://github.com/xivanalysis/xivanalysis/blob/dawntrail/src/data/STATUSES/layers/patch7.01.ts

So you'll just need to add a patch7.05 layer for STATUSES to cover the Magicked Swordplay duration change here.  And then fix Magicked Swordplay in the root file to be the 15000ms duration it had in 7.0 and 7.01.